<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FXProjectBrowser</class>
 <widget class="QWidget" name="FXProjectBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2067</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QFrame" name="frame_project">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_project">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_project">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tab_assets_shots">
      <property name="toolTip">
       <string>&lt;b&gt;Filter Assets&lt;/b&gt;&lt;hr&gt;Filter the assets by name.</string>
      </property>
      <property name="statusTip">
       <string>Filter the assets by name</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_assets">
       <attribute name="title">
        <string>Assets</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QFrame" name="frame_filter_assets">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_icon_filter_assets">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string>Icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_edit_filter_assets">
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_widget_assets">
          <property name="toolTip">
           <string>&lt;b&gt;Assets&lt;/b&gt;&lt;hr&gt;A tree of the existing assets.</string>
          </property>
          <property name="statusTip">
           <string>A tree of the existing assets</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">Assets</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_shots">
       <attribute name="title">
        <string>Shots</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QFrame" name="frame_filter_shots">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_icon_filter_shots">
             <property name="maximumSize">
              <size>
               <width>18</width>
               <height>18</height>
              </size>
             </property>
             <property name="text">
              <string>Icon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="line_edit_filter_shots">
             <property name="toolTip">
              <string>&lt;b&gt;Filter Shots&lt;/b&gt;&lt;hr&gt;Filter the shots by name.</string>
             </property>
             <property name="statusTip">
              <string>Filter the shots by name</string>
             </property>
             <property name="placeholderText">
              <string>Filter...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_widget_shots">
          <property name="toolTip">
           <string>&lt;b&gt;Shots&lt;/b&gt;&lt;hr&gt;A tree of the existing sequences and their shots.</string>
          </property>
          <property name="statusTip">
           <string>A tree of the existing sequences and their shots</string>
          </property>
          <property name="uniformRowHeights">
           <bool>false</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string notr="true">Shots</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitters_steps_tasks_info">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitters_steps_tasks">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="group_box_steps">
        <property name="title">
         <string>Steps</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QListWidget" name="list_steps">
           <property name="layoutMode">
            <enum>QListView::Batched</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="group_box_tasks">
        <property name="title">
         <string>Tasks</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="list_tasks">
           <property name="layoutMode">
            <enum>QListView::Batched</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>true</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="group_box_info">
       <property name="title">
        <string>Metadata</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="group_box_workfiles">
      <property name="title">
       <string>Workfiles</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkbox_display_latest_workfiles">
            <property name="text">
             <string>Show Only Latest</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>271</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_icon_filter_workfiles">
            <property name="maximumSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <property name="text">
             <string>Icon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="combobox_filter_workfiles">
            <item>
             <property name="text">
              <string>All</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Blender</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Houdini</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Maya</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Nuke</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Photoshop</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Substance Painter</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_widget_workfiles">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">Workfile</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Comment</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date Created</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Date Modified</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>User</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Size</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
